recipes/robotframework-requests/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "robotframework-requests" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "da7bf998f9cbf8261199db7c96b95be1bf29d1db7f59dfbc77c435761698dc75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - requests
+    - requests_ntlm
+    - robotframework
+
+test:
+  imports:
+    - RequestsLibrary
+
+about:
+  home: https://github.com/MarketSquare/robotframework-requests
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Could you add a comment about the `license_file` line that reads:

```
# license file not include in source distribution. See https://github.com/MarketSquare/robotframework-requests/pull/293
```

The [linked PR](https://github.com/MarketSquare/robotframework-requests/pull/293) is meant as a reminder to check to see if the license file is include in the source distribution in future releases to the package. Also note that upstream the file is called `LICENSE.md` not `LICENSE.txt`, so when it's time to go with the one that comes in the tarball, you'll need to change the `license_file` entry.

Reply:
Done! :)